lib/api.ts

```diff
@@ -1,7 +1,7 @@
 import { groq } from 'next-sanity';
 import { getClient } from '@lib/sanity';
 
-export const PAGE_TYPES = `['page', 'foundation', 'component', 'content', 'pattern', 'brand', 'resource', 'update']`;
+export const PAGE_TYPES = `['page', 'foundation', 'component', 'content', 'pattern', 'brand', 'resource', 'update', 'menus']`;
```

Comment:
I don't think we need 'menus' here unless you had a specific reason for adding it

Reply:
Oh no, i don't even know what this does